src/config.php

```diff
@@ -5,6 +5,8 @@
         'client' => env('SWISH_CLIENT_CERTIFICATE_PATH'),
         'password' => env('SWISH_CLIENT_CERTIFICATE_PASSWORD'),
         'root' => env('SWISH_ROOT_CERTIFICATE_PATH', true),
+        'signing' => env('SWISH_SIGNING_CERTIFICATE_PATH', null), // Optional, used for payouts
+        'signing_password' => env('SWISH_SIGNING_CERTIFICATE_PASSWORD', null), // Optional, used for payouts
     ],
     'endpoint' => \Olssonm\Swish\Client::PRODUCTION_ENDPOINT,
```

Comment:
@olssonm what do you think about adding this as a environment variable as well?

```suggestion
    'endpoint' => env('SWISH_URL', \Olssonm\Swish\Client::PRODUCTION_ENDPOINT),
```

Reply:
@vinkla Good call, definitely! Will add it in a separate PR soon, as Swish now also provides a staging environment that will be added as a constant.